content/src/controller/Controller.ts

```diff
@@ -568,6 +568,11 @@ export class Controller {
     // Path: /denylist/{type}/{id}
     // Body: JSON with ethAddress, signature and timestamp
 
+    if (this.denylist.isDisabled()) {
+      res.status(500).send()
```

Comment:
I didn't want to expose much information, so it would be visible whether or not the denylist is enabled.